### PR TITLE
CA-1454_viber_bot_public_doc

### PR DESCRIPTION
**Pull request by rumen-kyusakov**

### CODE REVIEW

File: docs/conversation/conversation-channel-support.md [hunk 1]
```diff
@@ -1686,17 +1701,352 @@ credentials configuration for **app** with SMS channel:
...
+
+2. Follow the steps in [Create Viber Bot Page](https://partners.viber.com/account/create-bot-account)
+to create your new bot. Once your bot is created its authentication token will appear in
+the account’s “edit info” [screen](https://partners.viber.com/).
+
+#### Channel Configuration
+
+The easiest way to configure your Conversation API **app** with Viber support is to use
+[Sinch Portal](https://dashboard.sinch.com/convapi/overview). Just select your **app** and
+click on "SET UP CHANNEL" beside the Viber Bot channel.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:1735]: maybe an extra thought here about "all you need is the token!"?

File: docs/conversation/conversation-channel-support.md [hunk 2]
```diff
@@ -1686,17 +1701,352 @@ credentials configuration for **app** with SMS channel:
...
+using the management API.
+
+##### Testing the integration
+
+Viber Bot API does not allow sending bot messages to Viber users which have not
+subscribed to the bot. For establishing subscription to a bot the Viber users must send a message to the bot.
+So to test your integration open the Viber app and search for the name of your Viber bot.
+Then send a message to it. You should receive two callbacks on the registered Conversation API webhooks -  
+one is `conversation_start_notification` and the other is the actual message send from the handset.
+Both callbacks contain a field `contact_id` which is the identifier of the
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:1780]: Im not sure if this is clear enough for the customer, maybe we can explicitly state that when the user starts the conversation with the bot the contact is automatically created in their App (do we mention that Viber contacts are App scoped by the way? we should!), and they dont need to create it

File: docs/conversation/conversation-channel-support.md [hunk 3]
```diff
@@ -1686,17 +1701,352 @@ credentials configuration for **app** with SMS channel:
...
+contact which was created for the Viber user. You can use this `contact_id`
+to reply to the Viber user.
+
+#### Rich Message Support
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:1784]: this should be a level above "Sending Messages" right?

File: docs/conversation/conversation-channel-support.md [hunk 4]
```diff
@@ -1686,17 +1701,352 @@ credentials configuration for **app** with SMS channel:
...
+    "media_message": {
+      "url": "https://1vxc0v12qhrm1e72gq1mmxkf-wpengine.netdna-ssl.com/wp-content/uploads/2018/12/favicon.png"
+    }
+  }
+}
+```
+
+The rendered message:
+
+![Media Message](images/channel-support/viber/viber_media.jpg)
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:1835]: My bad, I forgot the other media examples, I just added doc and video examples for Tranconding examples if you want to include them here too

File: docs/conversation/conversation-channel-support.md [hunk 5]
```diff
@@ -1686,17 +1701,352 @@ credentials configuration for **app** with SMS channel:
...
+https://whatsapp-adapter.{{REGION}}.conversation-api.int.prod.sinch.com/adapter/v1/{{CONVERSATION_APP_ID}}/callback
+```
 
 You also need to configure at least one Conversation API webhook which
 will trigger POST callbacks to the given URL.
-The triggers which are relevant for SMS channel are:
 
-- MESSAGE_DELIVERY - delivery receipts for SMS messages
-- MESSAGE_INBOUND - inbound messages e.g., contact replies
+### Viber Bot
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support.md:1716]: Sorry, definitely lot of extra work, but I see "Receiving messages" sections in a few channels above 😬

File: docs/conversation/conversation-channel-support.md (hunk 1 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-channel-support.md:1735]: added info on that

File: docs/conversation/conversation-channel-support.md (hunk 2 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-channel-support.md:1780]: addressed these I think

File: docs/conversation/conversation-channel-support.md (hunk 3 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-channel-support.md:1784]: well looking at the other channels it should be vice versa. Copied messenger setup and got that issue (and others :)) from there. Fixed for messenger too

File: docs/conversation/conversation-channel-support.md (hunk 4 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-channel-support.md:1835]: I do not want :)
No joke aside I think it is not needed - the document is already huge and we should probably spit it. Maybe we can add more examples when it is refactored

File: docs/conversation/conversation-channel-support.md (hunk 5 above)
Comment (rumen-kyusakov) [reply docs/conversation/conversation-channel-support.md:1716]: fixed

**Pull request merged**